docs/concepts/usage-ideas-examples.md

```diff
@@ -2,14 +2,317 @@
+
+### Music streaming platform
+
+[Audius](https://audius.co/) is a music streaming platform built on IPFS. Artists are in control, no fees are taken by the platform, and listeners can enjoy their music while offline.
+
+### Music player
+
+[Diffuse.sh](https://diffuse.sh/) is an online music player that you can connect to your music repositories to listen to it from anywhere. You can now connect it to IPFS repositories.
+
+## Decentralize the Web itself
+
+Some core pieces of the web are still centralized, making it more prone to break or being censored. IPFS can help and make the web more resilient.
+
+### Decentralized DNS
+
+The [Domain Name System](https://en.wikipedia.org/wiki/Domain_Name_System) (DNS) is one of the most centralized pieces of the web. By having to ask a central place to know where to find  `google.com` or `facebook.com`, it is a central point of failure. Backing up the DNS on IPFS could improve its availability. A prototype for Javascript has been [published on NPM](https://www.npmjs.com/package/orbitdns).
+
+### Archive the web
+
+Thanks to de-duplication, IPFS is a powerful tool for archiving the web. The [InterPlanetary Wayback](https://github.com/oduwsdl/ipwb) is tackling this endeavor.
+
+### Fight censorship
+
+Wikipedia has not been accessible in Turkey for a few years. The company behind IPFS, Protocol Labs, is hosting a mirror of Wikipedia on IPFS. See the original [blog post](https://blog.ipfs.io/24-uncensorable-wikipedia/) and the [project code](https://github.com/ipfs/distributed-wikipedia-mirror) for more information.
+
+### Alternative ISP
+
+[Althea](https://althea.net/about) is building an alternative [Internet Service Provider](https://en.wikipedia.org/wiki/Internet_service_provider) (ISP) in the US as the low level of competition brings the quality of service down and prices up. Althea especially targets rural areas, where the coverage by traditional ISP is low.
+
+## Blockchain use-cases
+
+IPFS is a natural fit for [blockchain](https://en.wikipedia.org/wiki/Blockchain) use cases. The common state of the chain is distributed on-chain among participants, and specific data is stored on IPFS. Thanks to content addressing, the blockchain only needs to store the IPFS multi-hash, and users are sure to fetch correct data from any of their peers. This architecture is becoming the de-facto standard for blockchain applications.
+
+### Global market place for data storage
+
+[Filecoin](https://filecoin.io/) lets any storage owner host data for users who need additional space. It is one of the reasons IPFS exists in the first place and will be a great accelerator of the IPFS growth once fully online later this year. The project is developed by [Protocol Labs](https://protocol.ai/).
+
+Textile.io is building the [Powergate](https://blog.textile.io/filecoin-developer-tools-concepts/) tool for your app to interact with Filecoin once the main net is launched.
+
+### Reconcile IoT and privacy
```

Comment:
IOTA is not about privacy. They want to entice devices into sharing data via micro-transactions, on the contrary.

Reply:
Could you expand on this section a bit further then? Looks like I've missed the point.

Reply:
I think there is confusion because there are 2 different companies: Iotex who wants to sell hardware and provide a platform for clients to keep their data safe. IOTA is a distributed ledger, but technically it's not a blockchain, but something else that they call the Tangle. 
I'll clarify.

Reply:
Done